nix/cardano/entrypoints.nix

```diff
@@ -6,6 +6,9 @@
   inherit (inputs.bitte-cells._writers.library) writeShellApplication;
   inherit (inputs.bitte-cells._utils.packages) srvaddr;
   inherit (cell) packages environments library constants;
+  packages' = packages;
+in nixpkgs.lib.makeOverridable ({ evalSystem ? throw "unreachable" }@args: let
+  packages = if args ? evalSystem then packages'.override { inherit evalSystem; } else packages';
 
```

Comment:
Won't this break any usage of `entrypoints` that doesn't override `evalSystem`?

Reply:
Nope, because then `args ? evalSystem` will evaluate to false and so we'll just call `package` without going through the `override` function.

Reply:
```
nix-repl> { a = throw "unreachable"; } ? a
true

nix-repl> ({ a ? throw "unreachable" }@args: args ? a) { }
false
```
Huh I didn't know that, it seems like throws are handled differently in `@args` expansion.

Reply:
Nope, nothing special about `throw`, it's just that `@`-binding binds *what was actually passed*, without any defaults if they were not provided.